orchagent/orch.cpp

```diff
@@ -685,6 +686,82 @@ Executor *Orch::getExecutor(string executorName)
     return NULL;
 }
 
+bool Orch::handleSaiCreateStatus(sai_api_t api, sai_status_t status)
```

Comment:
>status [](start = 61, length = 6)

Considering this is an interface design, and future user may have extended requirement. Is it reasonable to add another optional parameter like `void *context`, so user could pass any parameter they want and use it in their derived functions?

Reply:
Thanks for this comment. I added an optional parameter `void *context` to the functions.